zmon_worker_monitor/workflow.py

```diff
@@ -215,6 +215,9 @@ def process_message(queue, known_tasks, reactor, msg_obj, current_span):
     timelimit = msg_body.get('timelimit')  # [90, 60]
     t_hard, t_soft = timelimit
 
+    if taskname == 'trial_run':
```

Comment:
May be we can set a tag ``set_tag('task_name', taskname)``, that way we can search and isolate all operations/tasks.